src/Behat/Config/Formatter/FormatterConfigInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Formatter;
+
+use Behat\Config\ConfigInterface;
+
+interface FormatterConfigInterface extends ConfigInterface
```

Comment:
Should this interface include the `toArray()` function as well?

Reply:
The toArray function comes from the ConfigInterface, so yes.